### PR TITLE
Enhancements for Jumpbox Configuration, Bug Fixes on Cluster Deployment and Additional Instructions for Getting Started on Couchbase Server

### PR DESCRIPTION
**Pull request by cihanb**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@cihanb__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: couchbase-on-ubuntu/README.md [hunk 1]
```diff
@@ -1,33 +1,49 @@
 # Install a Couchbase cluster on Ubuntu Virtual Machines using Custom Script Linux Extension
```
Comment (singhkay) [inline couchbase-on-ubuntu/README.md]: Should be the original Azure location in the links below

File: couchbase-on-ubuntu/azuredeploy.json [hunk 2]
```diff
@@ -122,8 +123,8 @@
     },
     "sourceImageName": "b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-12_04_5_LTS-amd64-server-20150413-en-us-30GB",
     "sourceImage": "[concat('/',subscription().subscriptionId,'/services/images/',variables('sourceImageName'))]",
-    "templateBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/couchbase-on-ubuntu/",
-    "sharedBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/shared_scripts/ubuntu/",
+    "templateBaseUrl": "https://raw.githubusercontent.com/cihanb/azure-quickstart-templates/master/couchbase-on-ubuntu/",
```
Comment (singhkay) [inline couchbase-on-ubuntu/azuredeploy.json]: Should be the original Azure location in the links below

**Comment by singhkay** [comment]
@cihanb Left few comments. Few additional requests

* Can you also make sure all the file names are lower case?
* See our organization convention for multi deployment templates - https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/images/namingConvention.png, https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE

**Comment by cihanb** [comment]
Submitting a new one with corrections.

**Pull request closed without merging**